readme.html

```diff
@@ -53,5 +53,118 @@ <h2>Pagination</h2>
+    <p>
+        The app uses HTTP/ HTTPS protocol for communicating with the Okta server. Below are the default ports used by Splunk SOAR.
+        <table>
+            <tr class=plain>
+                <th>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Service Name</th>
+                <th>Transport Protocol</th>
+                <th>Port</th>
+            </tr>
+            <tr>
+                <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;http</td>
+                <td>tcp</td>
+                <td>80</td>
+            </tr>
+            <tr>
+                <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;https</td>
+                <td>tcp</td>
+                <td>443</td>
+            </tr>
+        </table>
+    </p>
+    <p>
+        <h2>Permissions</h2>
+        <p>
+            Super admins can assign admin permissions to individuals or groups.
+            <br>
+            <b>Individual</b> assignments are better for a manageable number of admin accounts. When you assign admin permissions to individuals, you do so one at a time, whenever necessary. Remember that every org must always have at least one individually assigned super admin.
+            <br>
+            <b>Admin groups</b> work better when you need to onboard a large number of admins quickly. Assign those admins to one group, and then grant admin permissions to that group. Okta groups, AD groups, and LDAP groups are all eligible.
+            <br><br>
+            To assign the roles to any user, it is required to run the <b>"assign role"</b> action using the asset configured with the API token of a <b>Super admin user(SUPER_ADMIN)</b> in the OKTA server to assign appropriate permissions to that normal user.
+            <br><br>
+            Step to assign a Super admin role to user or group.
+            <br>
+            <ul>
+                <li>1. In the Admin Console, go to <b>Security > Administrators</b>.
+                </li>
+                <li>
+                    2. Click <b>Add Administrator</b>, depending on whether you are assigning privileges to an individual or group.
+                </li>
+                <li>
+                    3. In the <b>Grant administrator role to</b> field, begin typing the name of the user that we want to assign admin privileges to and select the correct user from the search list.
+                </li>
+                <li>
+                    4. Select the <b>Super Administrator</b> roles you want the user or group to have. You can assign multiple admin roles to an individual or group.
+                </li>
+                <li>
+                    5. Some admin roles require additional input. If you selected Application Administrator, Group Administrator, Help Desk Administrator, or Group Membership Administrator, you need to indicate whether that role administers all users, groups, or apps, or just specific ones. (Copy and Paste commands allow you to apply the same assignments to multiple roles).
```

Comment:
can we remove extra `or` from role administers all users, groups, `or` apps, or just specific ones.

Reply:
Removed extra `or`.